app/models/comment.rb

```diff
@@ -0,0 +1,14 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Here also, the association for the comment should be using a `author_id` column. Hence, Please apply `belongs_to :author, class_name: 'User'`